### PR TITLE
Update the version of jupyterlab manually

### PR DESCRIPTION
**Pull request by dsblank**
I tried a variety of methods of updating jupyterlab, and this finally worked.

You may need to adjust the text appropriately.

An error occurred when trying to build the `jupyterlab_apod` extension and needed jupyterlab version updated.

### CODE REVIEW

**Comment by dsblank** [comment]
Bah. This didn't fix the issue because I had an older version of jupyterlab installed via pip. The real solution was to `pip uninstall jupyterlab`. Perhaps a different note should be added in these docs.

**Comment by dsblank** [comment]
I still can't get the demo to build:

```
$ jupyter labextension install . --no-build
Node v10.8.0

> /home/dblank/miniconda3/envs/jupyterlab-ext/bin/npm pack /home/dblank/jupyterlab_apod

> jupyterlab_apod@0.1.0 prepare /home/dblank/jupyterlab_apod
> npm run clean && npm run build


> jupyterlab_apod@0.1.0 clean /home/dblank/jupyterlab_apod
> rimraf lib && rimraf tsconfig.tsbuildinfo


> jupyterlab_apod@0.1.0 build /home/dblank/jupyterlab_apod
> tsc

npm notice 
npm notice 📦  jupyterlab_apod@0.1.0
npm notice === Tarball Contents === 
npm notice 1.1kB package.json   
npm notice 436B  README.md      
npm notice 211B  lib/index.d.ts 
npm notice 269B  lib/index.js   
npm notice 0     style/index.css
npm notice === Tarball Details === 
npm notice name:          jupyterlab_apod                         
npm notice version:       0.1.0                                   
npm notice filename:      jupyterlab_apod-0.1.0.tgz               
npm notice package size:  1.0 kB                                  
npm notice unpacked size: 2.0 kB                                  
npm notice shasum:        fcf3a77398a93359622008fcc48e4e470d68a466
npm notice integrity:     sha512-AAs5dxshsA+uX[...]Sqo22jmF39jKA==
npm notice total files:   5                                       
npm notice 
jupyterlab_apod-0.1.0.tgz

Errored, use --debug for full output:
ValueError: 
"jupyterlab_apod@0.1.0" is not compatible with the current JupyterLab
Conflicting Dependencies:
JupyterLab              Extension        Package
>=0.19.1 <0.20.0        >=1.0.0 <2.0.0   @jupyterlab/application
```

**Comment by dsblank** [comment]
Success! I had to remove the pip version (as per above) and manually update jupyterlab (as per above). Then I have a working extension.

File: docs/source/developer/extension_tutorial.rst [hunk 1]
```diff
@@ -76,6 +76,12 @@ Note: You'll need to run the command above in each new terminal you open
 before you can work with the tools you installed in the
 ``jupyterlab-ext`` environment.
 
+Note: if you have an older version of jupyterlab previously installed, you may need to update
+the version of jupyterlab manually.
+
+.. code:: bash
+    conda install -c conda-forge jupyterlab=1.0.2
```
Comment (jasongrout) [inline docs/source/developer/extension_tutorial.rst]:
```suggestion
    conda install -c conda-forge jupyterlab=1
```

File: docs/source/developer/extension_tutorial.rst [hunk 2]
```diff
@@ -76,6 +76,12 @@ Note: You'll need to run the command above in each new terminal you open
 before you can work with the tools you installed in the
 ``jupyterlab-ext`` environment.
 
+Note: if you have an older version of jupyterlab previously installed, you may need to update
```
Comment (jasongrout) [inline docs/source/developer/extension_tutorial.rst]:
```suggestion
Note: if you have an older version of JupyterLab previously installed, you may need to update
```

File: docs/source/developer/extension_tutorial.rst [hunk 3]
```diff
@@ -76,6 +76,12 @@ Note: You'll need to run the command above in each new terminal you open
...
+the version of jupyterlab manually.
```
Comment (jasongrout) [inline docs/source/developer/extension_tutorial.rst]:
```suggestion
the version of JupyterLab manually.
```

**Comment by dsblank** [comment]
Thanks!

**Comment by jasongrout** [comment]
Looks like there is an error building the docs:

/home/vsts/work/1/s/docs/source/developer/extension_tutorial.rst:82: WARNING: Error in "code" directive:
maximum 1 argument(s) allowed, 6 supplied.

**Comment by dsblank** [comment]
> Looks like there is an error building the docs:

Should be fixed now.

**Comment by jasongrout** [comment]
Thanks!

**Pull request merged**